kratos/processes/calculate_discontinuous_distance_to_skin_process.cpp

```diff
@@ -542,6 +659,216 @@ namespace Kratos
 		MathUtils<double>::CrossProduct(rIntObjNormal, rGeometry[1]-rGeometry[0], rGeometry[2]-rGeometry[0]);
 	}
 
+	template<std::size_t TDim>
+	void CalculateDiscontinuousDistanceToSkinProcess<TDim>::ComputeExtrapolatedEdgesIntersectionsIfIncised(
+        const Element& rElement,
+		unsigned int &rNumCutEdges,
+		array_1d<double, (TDim == 2) ? 3 : 6>& rCutEdgesRatioVector,
+		array_1d<double,3> &rExtraGeomNormal,
+		array_1d<double, (TDim == 2) ? 3 : 6>& rCutExtraEdgesRatioVector)
+	{
+		if (TDim == 2 && rNumCutEdges == 1) {
+			ComputeExtrapolatedGeometryIntersections(rElement, rNumCutEdges, rCutEdgesRatioVector, rExtraGeomNormal, rCutExtraEdgesRatioVector);
+		} else if (TDim == 3) {
+			if (rNumCutEdges == 3) {
+				// if all three cut edges share one node, then the element is intersected and not incised
+				if (CheckIfCutEdgesShareNode(rElement, rCutEdgesRatioVector)) {
+					return;
+				}
+				ComputeExtrapolatedGeometryIntersections(rElement, rNumCutEdges, rCutEdgesRatioVector, rExtraGeomNormal, rCutExtraEdgesRatioVector);
+			} else if (rNumCutEdges == 1 || rNumCutEdges == 2) {
+				ComputeExtrapolatedGeometryIntersections(rElement, rNumCutEdges, rCutEdgesRatioVector, rExtraGeomNormal, rCutExtraEdgesRatioVector);
+			}
+		}
+	}
+
+	template<std::size_t TDim>
+	void CalculateDiscontinuousDistanceToSkinProcess<TDim>::ComputeExtrapolatedGeometryIntersections(
+        const Element& rElement,
+		unsigned int& rNumCutEdges,
+		array_1d<double, (TDim == 2) ? 3 : 6>& rCutEdgesRatioVector,
+		array_1d<double,3>& rExtraGeomNormal,
+		array_1d<double, (TDim == 2) ? 3 : 6>& rCutExtraEdgesRatioVector)
```

Comment:
```suggestion
	void CalculateDiscontinuousDistanceToSkinProcess<TDim>::ComputeExtrapolatedGeometryIntersections(
	    const Element& rElement,
            unsigned int& rNumCutEdges,
            array_1d<double, (TDim == 2) ? 3 : 6>& rCutEdgesRatioVector,
            array_1d<double,3>& rExtraGeomNormal,
            array_1d<double, (TDim == 2) ? 3 : 6>& rCutExtraEdgesRatioVector)
```
Minor format issue. I think this is also happening in other places (I guess you have configured your editor to use tabs instead of spaces).

Reply:
Could very well be, I'll check and change that!

Reply:
It's set to spaces.. it replaces a tab with 4 spaces

Reply:
Thanks. This is what we decided in the [style guide](https://github.com/KratosMultiphysics/Kratos/wiki/Style-Guide)